flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/ScheduledNotificationReceiver.java

```diff
@@ -50,26 +50,26 @@ public void onReceive(final Context context, Intent intent) {
       notification.when = System.currentTimeMillis();
       NotificationManagerCompat notificationManager = NotificationManagerCompat.from(context);
       notificationManager.notify(notificationId, notification);
-      boolean repeat = intent.getBooleanExtra("repeat", false);
-      if (!repeat) {
+      if (!intent.getBooleanExtra("repeat", false)) {
         FlutterLocalNotificationsPlugin.removeNotificationFromCache(context, notificationId);
       }
     } else {
-      Gson gson = FlutterLocalNotificationsPlugin.buildGson();
-      Type type = new TypeToken<NotificationDetails>() {}.getType();
+      final Gson gson = FlutterLocalNotificationsPlugin.buildGson();
+      final Type type = new TypeToken<NotificationDetails>() {}.getType();
```

Comment:
While I agree with all these changes, we try to keep the changes from our branch the https://github.com/MaikuB/flutter_local_notifications/tree/master to a minimum.
Because we constantly merge the main projects changes into our, it is much easier if they are as similar as possible

So could you revert the changes that is not strictly necessary?

Reply:
Yes, that makes sense.